imapautofiler/tests/base.py

```diff
@@ -54,6 +54,12 @@ def construct_message(headers):
     'Subject': 'Re: ответ на предыдущее сообщение',
 })
 
+date = time.ctime(time.time())
+RECENT_MESSAGE = MESSAGE.copy()
+RECENT_MESSAGE.update({
+    'Date': '{}'.format(date),
```

Comment:
Does this string formatting produce the same results as email.utils.format_datetime()? 

https://docs.python.org/3.5/library/email.util.html#email.utils.format_datetime